src/main/java/com/configcat/Setting.java

```diff
@@ -1,64 +1,92 @@
 package com.configcat;
 
-import com.google.gson.JsonElement;
 import com.google.gson.annotations.SerializedName;
 
+/**
+ * Feature flag or setting.
+ */
 public class Setting {
-    /**
-     * Value of the feature flag / setting.
-     */
-    @SerializedName(value = "v")
-    private JsonElement value;
 
     /**
-     * Type of the feature flag / setting.
+     * Setting type.
```

Comment:
I'd rather move these docs comments from private fields to the public accessors.